src/baseonnx/read.jl

```diff
@@ -8,49 +8,49 @@ function array(p::TensorProto, wrap=Array)
     # Copy pasted from jl
     # Can probably be cleaned up a bit 
     # TODO: Add missing datatypes...
-    if p.data_type === TensorProto_DataType.INT64
+    if p.data_type === var"TensorProto.DataType".INT64
```

Comment:
Could we make this less hideous looking by having something like
```julia
const TensorProto_DataType = var"TensorProto.DataType"
```
in BaseOnnx.jl?

Reply:
Sure, I took your suggestion in the latest commits.